src/containers/Articles/Article/CreateArticle.jsx

```diff
@@ -0,0 +1,159 @@
+import React, { Component } from "react";
+import { connect } from "react-redux";
+import { EditorState } from "draft-js";
+import { Redirect } from "react-router-dom";
+
+import * as actions from "../../../store/actions/index";
+import CreateArticleForm from "../../../components/UI/Article/ArticleForm";
+import { successToast } from "../../../components/Toast/toast";
+
+export class CreateArticle extends Component {
+
+    state = {
+        editorState: EditorState.createEmpty(),
+        articleData: {
+            title: "",
+            description: "",
+            body: "",
+            image: null
+        },
+        error: false,
+        titleTouched: false,
+        descriptionTouched: false,
+        bodyTouched: false,
+        disabled: false,
+        picture: null
+    };
+
+    onChangeHandler = (editorState) => {
+        this.setState({editorState});
+    };
+
+    bodyChangedHandler=( event, editor ) => {
+        const data = editor.getData();
+        const currentState = {...this.state};
+        const articleData = {...currentState.articleData};
+        articleData.body = data;
+        let disableBtn = true;
+        if ( !articleData.body
+            || articleData.body.length < 5 ||
+            !articleData.title || !articleData.description) {
+            disableBtn = false
+        };
+
+        this.setState({
+            articleData: articleData,
+            disabled: disableBtn
+        });
+    }
+
+
+    inputHandler = (e) => {
+        const currentState = {...this.state};
+        const articleData = {...currentState.articleData};
```

Comment:
It is a repetition on line `35` and it can be made better by extracting exact values need as described in the comment  on line `35`